include/wlr/render/egl.h

```diff
@@ -21,6 +21,7 @@ struct wlr_egl {
 		bool dmabuf_import;
 		bool dmabuf_import_modifiers;
 		bool bind_wayland_display;
+		bool context_priority;
```

Comment:
Why does this need to be stored?

Reply:
It's just for consistency with other extensions. Can remove it if you don't like it.

Reply:
There are other extensions we're using which aren't listed here, so the whole thing is kind of inconsistent.

Reply:
Added a more consistent EGL extension checking (GL extension checking doesn't happen here).

Reply:
-EGL_KHR_debug
-EGL_(EXT/KHR)_platform_wayland
-EGL_(EXT/KHR)_platform_x11
-EGL_(MESA/KHR)_platform_gbm
-EGL_MESA_platform_surfaceless
-EGL_EXT_platform_base (Although we fail if this doesn't exist)

Reply:
Ah, right, I missed a few it seems.

Not sure how to check `EGL_KHR_debug`: we use it before initializing the display.

Reply:
For all platforms, it's probably better to expose a `wlr_egl_check_extension` function.

Reply:
It's not something that I actually care about that much, but I was really just pointing out the inconsistency.
Most of these things will just fail right at the start if they don't exist, and we have no real reason to remember them.

Reply:
Okay, probably better to remember only what we need to remember then.